app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+# User model class for all user types.
+class User < ApplicationRecord
```

Comment:
I had to Google this since I had no idea what `ApplicationRecord` was, although since the specs were passing I assumed it made sense. Guess I'd better read up on Rails 5 :stuck_out_tongue:.